src/cxy_ctl.erl

```diff
@@ -58,14 +61,43 @@ make_process_dictionary_default_value(Key, Value) ->
 
 %% Use a raw tuple because two record structs with the same key would
 %% be needed, or redundant record name in key position.
+
+%% Cxy counters tuple...
 -define(MAX_PROCS_POS,    2).
 -define(ACTIVE_PROCS_POS, 3).
 -define(MAX_HISTORY_POS,  4).
 
-make_proc_values(Task_Type, Max_Procs_Allowed, Max_History) ->
+%% Cumulative performance moving average tuple...
+-define(CMA_TYPE_POS,        1).
+-define(CMA_SPAWN_CMA_POS,   2).
+-define(CMA_EXEC_CMA_POS,    3).
+-define(CMA_RING_SIZE_POS,   4).
+-define(CMA_SLOW_FACTOR_POS, 5).
+
+%% Read all values using update_counter to avoid releasing write_lock for a read_lock.
+-define(CMA_READ_CMD,
+        [{?CMA_SPAWN_CMA_POS,   0},
+         {?CMA_EXEC_CMA_POS,    0},
+         {?CMA_RING_SIZE_POS,   0},
+         {?CMA_SLOW_FACTOR_POS, 0}
+        ]).
```

Comment:
I am trying to avoid constructing a record. I would have to give it a name so it would be {cma, {cam, type}, ...} in the ets table. Considering lots and lots of these, I don't need to waste that space. It is a library for large buffers.

Previously it was only the one tuple, now there are two, but still... it is a space-saving decision.